content/en/docs/reference/config/security/authorization-policy/index.html

```diff
@@ -86,6 +86,25 @@
     - operation:
         methods: [&quot;POST&quot;]
 </code></pre>
+<p>The following is another example that sets <code>action</code> to &ldquo;DENY&rdquo; to create a deny policy.
+  It denies requests all the &ldquo;POST&rdquo; method request on port &ldquo;8080&rdquo; on all workloads
+  in the &ldquo;foo&rdquo; namespace.</p>
+  <pre><code class="language-yaml">apiVersion: security.istio.io/v1beta1
+  kind: AuthorizationPolicy
+  metadata:
+    name: httpbin
+    namespace: foo
+  spec:
+    action: DENY
+    rules:
+    - to:
+      - operation:
+          methods: [&quot;POST&quot;]
+          ports: [&quot;8080&quot;]
+  </code></pre>
+<p>If we remove the ports from the rules of the above-mentioned deny policy, then it will just have the http attribute in the rules and
+it will deny all the traffic on raw tcp ports. It's better to always add ports in the deny policy if you don't want to deny all 
+the TCP traffic.<a href="https://istio.io/latest/docs/tasks/security/authorization/authz-tcp/">Authorization Policy for TCP Ports </a></p> 
```

Comment:
This change needs to be made in the istio/api repo. See the top of the file. The nightly run will overwrite this change with what's there.